task-manager/api/util/tasks.js

```diff
@@ -0,0 +1,44 @@
+const validaDescricao = (descricao) => {
+  const tamanho = descricao.length;
+  if (tamanho <= 5) {
+    throw 'Descricao invalida';
+  }
+};
+const isValidDate = (...date) => {
```

Comment:
Opa, lembra que conversamos sobre o nome desse método? Que na real o que ele está fazendo é validar múltiplas datas e esse nome acaba gerando confusão.

O ideal seria trocar ele para validar apenas uma data e chamar ele múltiplas vezes fora OU, se isso não for possível, renomeá-lo pra deixar mais claro sua utilização

Reply:
feito.

Reply:
:+1: Não consigo marcar como resolved, mas por mim pode fechar!